lms/djangoapps/teams/views.py

```diff
@@ -106,6 +118,24 @@ def is_feature_enabled(course):
     return settings.FEATURES.get('ENABLE_TEAMS', False) and course.teams_enabled
 
 
+def has_discussion_privileges(user, course_id):
```

Comment:
Get @dianakhuang's version of this method (which lives in django_comment_client, not here).